lemur/plugins/lemur_digicert/tests/test_digicert.py

```diff
@@ -35,14 +35,47 @@ def test_map_fields(app):
             'signature_hash': 'sha256'
         },
         'organization': {'id': 111111},
-        'custom_expiration_date': arrow.get(2017, 5, 7).format('YYYY-MM-DD')
+        'custom_expiration_date': arrow.get(2017, 5, 7).format('YYYY-MM-DD'),
+        'validity_years': 1
+    }
+
+
+def test_map_fields_with_validity_years(app):
+    from lemur.plugins.lemur_digicert.plugin import map_fields
+
+    names = [u'one.example.com', u'two.example.com', u'three.example.com']
```

Comment:
Ah, I see this is a python2.7 issue. That sounds reasonable for the 0.4 branch (as python 2.7) as it's still supported there.